---
title: 使用redis-py客户端时连接集群时，报错：ERROR sending 'config get cluster-require-full-coverage' command to redis server
date: 2022-01-12 14:33:08
permalink: /pages/477067/
article: true
---

## 问题原因

出现这种异常的原因是云数据库Redis集群限制了`config`命令的使用，导致将“config get cluster require full coverage”命令发送到redis服务器时出错。

## 解决方案

在连接Redis集群的代码中配置跳过检查的参数`skip_full_coverage_check=True`就可以了。

示例如下：

```python
redis_conn= RedisCluster(startup_nodes=startup_nodes, password="XXXXXXXXXXXXXXX",skip_full_coverage_check=True,decode_responses=True)
```

